contracts/examples/Create2OwnableFactory.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.0;
+
+import "../utils/Create2.sol";
+import "../ownership/Ownable.sol";
+
+contract Create2OwnableFactory {
+  function deploy(bytes32 salt, bytes memory code) public {
+      address deployedContract = Create2.computeAddress(salt, code);
+      Create2.deploy(salt, code);
+      Ownable(deployedContract).transferOwnership(msg.sender);
+  }
+  function computeAddress(
+      bytes32 salt, bytes memory code
+  ) public view returns (address) {
+      return Create2.computeAddress(salt, code);
+  }
+}
```

Comment:
In the contract `Create2.sol`, you have two approach named `computeAddress()`, so is there a necessary to add the other function 
```javascript
function computeAddress(
      address deployer, bytes32 salt, bytes memory code
  ) public view returns (address) {
      return Create2.computeAddress(deployer, salt, code);
  }
```